src/visualizations/config/adapters/dhis_highcharts/chart.js

```diff
@@ -31,17 +34,30 @@ const getEvents = () => ({
                     })
                 }
             })
+            renderCustomSVG.call(this, visType)
         },
     },
 })
 
-export default function (layout, el, dashboard) {
+export default function (layout, el, extraOptions, series) {
     return Object.assign(
         {},
         getType(layout.type),
         { renderTo: el || layout.el },
         DEFAULT_CHART,
-        dashboard ? DASHBOARD_CHART : undefined,
-        getEvents()
+        extraOptions.dashboard ? DASHBOARD_CHART : undefined,
+        getEvents(layout.type),
+        layout.type === VIS_TYPE_SINGLE_VALUE
```

Comment:
These will eventually need to be moved to a separate file when a new custom SVG type comes along...

Reply:
Code was refactored in [159727c](https://github.com/dhis2/analytics/pull/1698/commits/159727c42ea6c6c999b7a66f4280752f79e6b9dc) to address this